queries/java/highlights.scm

```diff
@@ -0,0 +1,147 @@
+
+; Methods
+
+(method_declaration
+  name: (identifier) @function.method)
+(method_invocation
+  name: (identifier) @function.method)
+(super) @function.builtin
+
+; Annotations
+
+(annotation
+  name: (identifier) @attribute)
+(marker_annotation
+  name: (identifier) @attribute)
+
+"@" @operator
+
+; Types
+
+(interface_declaration
+  name: (identifier) @type)
+(class_declaration
+  name: (identifier) @type)
+(enum_declaration
+  name: (identifier) @type)
+
+((field_access
+  object: (identifier) @type)
+ (#match? @type "^[A-Z]"))
+((scoped_identifier
+  scope: (identifier) @type)
+ (#match? @type "^[A-Z]"))
+
+(constructor_declaration
+  name: (identifier) @type)
+
+(type_identifier) @type
+(boolean_type) @type.builtin
+(integral_type) @type.builtin
+(floating_point_type) @type.builtin
+(floating_point_type) @type.builtin
+(void_type) @type.builtin
+
+; Variables
+
+((identifier) @constant
+ (#match? @constant "^_*[A-Z][A-Z\d_]+"))
+
+(identifier) @variable
+
+(this) @variable.builtin
+
+; Literals
+
+(hex_integer_literal) @number
+(decimal_integer_literal) @number
+(octal_integer_literal) @number
+(decimal_floating_point_literal) @number
+(hex_floating_point_literal) @number
+(character_literal) @string
+(string_literal) @string
+(null_literal) @constant.builtin
+
+(comment) @comment
+
+[
+"true"
+"false"
+] @boolean
+
```

Comment:
Does Neovim accept your queries?

For me this has to be changed to
```
[
 (true)
 (false)
] @boolean
```

Reply:
I haven't tested it because I only have a windows laptop rn. I'll test it tomorrow, thanks

Reply:
I changed it anyway. Since you seem to have tested it, does highlighting work?